components/board/pinwrappers/digital_interrupts.go

```diff
@@ -84,9 +74,10 @@ func AddCallback(i *BasicDigitalInterrupt, c chan board.Tick) {
 }
 
 // RemoveCallback removes a listener for interrupts.
-func (i *BasicDigitalInterrupt) RemoveCallback(c chan board.Tick) {
+func RemoveCallback(i *BasicDigitalInterrupt, c chan board.Tick) {
```

Comment:
Discussion question: if this only works on `BasicDigitalInterrupts`, maybe it should remain a member function in that class, instead of a standalone function. I guess I'm open to either possibility, though my gut instinct is that it belongs in the class.